explorer/scripts/healthcare_gov_etl.py

```diff
@@ -1,144 +0,0 @@
-#!/usr/local/bin/python3.6
```

Comment:
maybe I should remove this. or make it more clear that it's redundant

Reply:
Remove if you think it won't need to be used. We can always go back in git if needed

Reply:
this was actually just renamed and I failed to use git mv. just added it again under a different name. I think it's still useful